openssl-dynamic/src/main/c/ssl.c

```diff
@@ -46,6 +51,10 @@ extern apr_pool_t *tcn_global_pool;
 ENGINE *tcn_ssl_engine = NULL;
 void *SSL_temp_keys[SSL_TMP_KEY_MAX];
 
+#ifndef OPENSSL_NO_ENGINE
+UI_METHOD *ui_method = NULL;
```

Comment:
should this be defined as `static`?

Reply:
Can’t hurt... let me also make tcn_ssl_engine static while I am here

Reply:
(static will prevent the variable from being visible/exported from this file).